infra/helper.py

```diff
@@ -419,6 +462,117 @@ def check_build(args):
   return exit_code
 
 
+def _get_fuzz_targets(project_name):
+  """Return names of fuzz targest build in the project's /out directory."""
+  command = [
+      'find',
+      _get_output_dir(project_name),
+      '-maxdepth',
+      '1',
+      '-type',
+      'f',
+      '-executable'
+  ]
+  output = subprocess.check_output(command)
+  return [os.path.basename(path) for path in output.split()]
+
+
+def _get_latest_corpus(project_name, fuzz_target, base_corpus_dir):
+  """Download the latest corpus for the given fuzz target."""
+  corpus_dir = os.path.join(base_corpus_dir, fuzz_target)
```

Comment:
I'm not sure if the script is meant to be compatible with `python3`, but this line seems to cause `python3` to fail with `Can't mix strings and bytes in path components`:
```sh
# python3 ./infra/helper.py profile systemd
Downloading corpus for systemd
Traceback (most recent call last):
  File "./infra/helper.py", line 755, in <module>
    sys.exit(main())
  File "./infra/helper.py", line 165, in main
    return profile(args)
  File "./infra/helper.py", line 550, in profile
    download_corpus(args.project_name)
  File "./infra/helper.py", line 542, in download_corpus
    thread_pool.map(_download_for_single_target, fuzz_targets)
  File "/usr/lib64/python3.6/multiprocessing/pool.py", line 266, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/usr/lib64/python3.6/multiprocessing/pool.py", line 644, in get
    raise self._value
  File "/usr/lib64/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib64/python3.6/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "./infra/helper.py", line 538, in _download_for_single_target
    _get_latest_corpus(project_name, fuzz_target, corpus_dir)
  File "./infra/helper.py", line 482, in _get_latest_corpus
    corpus_dir = os.path.join(base_corpus_dir, fuzz_target)
  File "/usr/lib64/python3.6/posixpath.py", line 94, in join
    genericpath._check_arg_types('join', a, *p)
  File "/usr/lib64/python3.6/genericpath.py", line 151, in _check_arg_types
    raise TypeError("Can't mix strings and bytes in path components") from None
TypeError: Can't mix strings and bytes in path components
```

Reply:
The script is not compatible with python3 yet. @alex has also suggested that in https://github.com/google/oss-fuzz/pull/1494#discussion_r194912592, so I guess we should file a bug at least